src/sonic-host-services/scripts/check_system

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+echo "check_system called"
+SYSTEM_ERR=4
+
+# touch /var/log/syslog
+# ret=$?
+
+# to help simulate
+if [ -f /var/log/read_only ]; then
```

Comment:
how can this check system read-only?

Reply:
if the disk become read-only, how can this file be generated?

Reply:
This is a hack for immediate testing. The commented lines above are the actual code.

Reply:
Removed hack.